### ISSUE TITLE
There are white borders when the video player is in fullscreen mode.

### ISSUE THREAD

**Issue by tangxiaoqi-tangxiao** [huynhsontung/Screenbox#329]
**Describe the bug**

When a video finishes playing (in a paused state), there is a severe white border that appears in fullscreen mode

**How To Reproduce**

**Expected behavior**

**Screenshots**

![image](https://github.com/huynhsontung/Screenbox/assets/54069236/7cb1b3b3-403f-433d-aace-5eaf598ca88a)

**Environment**

 - OS: [e.g. Windows 10, Windows 11, Xbox]
 - Device: [e.g. PC, Laptop model, Xbox model]
 - App Version [e.g. v0.9.15]

**Additional context**

**Comment by United600** [comment]
Is the app version that you provided correct?

Either way, I can somewhat reproduce the issue. To reproduce it, let a video finish on window mode and then toggle the fullscreen mode.

https://github.com/huynhsontung/Screenbox/assets/698155/8c8039f3-3447-41ea-97c9-cb7501e5ab33

**Comment by huynhsontung** [comment]
This is a VLC limitation I think. When the playback is finished, VLC stops sending new frames to the panel so what you see after is just whatever is in the last frame buffer and cannot adapt to window size changes. The only option available from our end is to clear the frame when playback finishes with make the whole panel transparent/black, which I don't think is desirable.

**Comment by United600** [comment]
> This is a VLC limitation I think. When the playback is finished, VLC stops sending new frames to the panel so what you see after is just whatever is in the last frame buffer and cannot adapt to window size changes. The only option available from our end is to clear the frame when playback finishes with make the whole panel transparent/black, which I don't think is desirable.

Yeah this is a edge case, I guess that's why VLC UWP clears/closes the playback.
